crates/turborepo-lib/src/commands/daemon.rs

```diff
@@ -123,10 +123,14 @@ pub async fn daemon_client(command: &DaemonCommand, base: &CommandBase) -> Resul
             }
         }
         DaemonCommand::Logs => {
-            let mut client = connector.connect().await?;
-            let status = client.status().await?;
-            let log_file = log_filename(&status.log_file)?;
+            let log_file = if let Ok(log_file) = get_log_file_from_daemon(connector).await {
+                log_file
+            } else {
+                get_log_file_from_folder(base).await?
```

Comment:
Do we need the fallback?

Reply:
I found it really annoying that you can't see logs when the daemon has crashed

Reply:
I think I commented on the wrong line. I meant, do we need the global location ?